### PR TITLE
Merge and multitarget samples, target .NET 4.6.1 and above, adopt Community.Mvvm instead of MvvmLight

### PR DESCRIPTION
**Pull request by virzak**
# Description

- Remove support for .Net Frameworks before 4.6.1. Ookii.Dialogs.Wpf supports 4.6.2 and above and so do mvvm-dialog samples.
- Reduce the number of sample projects by multitargeting sample projects to both .Net Core and .Net Framework.
- Replace MVVMLight with [CommunityToolkit.Mvvm](https://github.com/CommunityToolkit/dotnet/tree/main/CommunityToolkit.Mvvm)

Part of #177 

# Checklist

- [x] I have squashed my commits into a single one with a message that aligns with the contributing guidelines
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by virzak** [comment]
@FantasticFiasco all good, no rush.

File: MvvmDialogs.nuspec [hunk 1]
```diff
@@ -15,45 +15,22 @@
...
-    <file src="src\net\bin\Release\net472\MvvmDialogs.pdb" target="lib\net472\MvvmDialogs.pdb" />
+    <file src="src\net\bin\Release\net461\MvvmDialogs.dll" target="lib\net461\MvvmDialogs.dll" />
+    <file src="src\net\bin\Release\net461\MvvmDialogs.xml" target="lib\net461\MvvmDialogs.xml" />
+    <file src="src\net\bin\Release\net461\MvvmDialogs.pdb" target="lib\net461\MvvmDialogs.pdb" />
 
     <!-- .NET Core -->
     <file src="src\net\bin\Release\netcoreapp3.1\MvvmDialogs.dll" target="lib\netcoreapp3.1\MvvmDialogs.dll" />
     <file src="src\net\bin\Release\netcoreapp3.1\MvvmDialogs.xml" target="lib\netcoreapp3.1\MvvmDialogs.xml" />
     <file src="src\net\bin\Release\netcoreapp3.1\MvvmDialogs.pdb" target="lib\netcoreapp3.1\MvvmDialogs.pdb" />
-    <file src="src\net\bin\Release\net5.0-windows\MvvmDialogs.dll" target="lib\net5.0-windows7.0\MvvmDialogs.dll" />
```
Comment (FantasticFiasco) [inline MvvmDialogs.nuspec:54]: Please remind me, why are we removing support for `net5.0-windows`?

**Comment by FantasticFiasco** [comment]
A todo list for myself.

- [ ] Remove `MvvmLight` dependency from UWP sample projects

**Comment by FantasticFiasco** [comment]
@virzak thanks for the contribution!

**Comment by virzak** [comment]
@FantasticFiasco Looks like it created a bit more work for you than I expected, but glad it worked out :)

**Comment by FantasticFiasco** [comment]
No worries. What we have now is better than what we had before your PR, and that's because of you.

**Pull request merged**